### DISCUSSION TITLE
Is it possible to render something like this?

### DISCUSSION THREAD

**Discussion by nyngwang** [goolord/alpha-nvim#223] [Q&A]
![image](https://github.com/goolord/alpha-nvim/assets/24765272/ada67912-5ff5-4202-920d-478d7c415a56)

but when I copy the raw text returned by `curl` I got `^[`(escape-sequence? of `<Esc>` key)

**Comment by goolord** [comment]
there is a `terminal` type documented here https://github.com/goolord/alpha-nvim/blob/1838ae926e8d49fe5330d1498ee8289ae2c340bc/doc/alpha.txt#L166 that uses a floating terminal window to display the output of a command. you could pipe this output to a file and `cat` it. 

alternatively, you could write a script that converts ansi escape codes into neovim highlights or something

**Reply by nyngwang** [reply]
Thanks for your help. I was thinking about whether it's possible to render text like this into a normal buffer instead of a terminal one. What did you mean by converting ANSI escape codes into Neovim highlights?

**Reply by Asthestarsfalll** [reply]
@nyngwang Hi.
You may refer to [here](https://github.com/Asthestarsfalll/img2art?tab=readme-ov-file#experimental-feature) to convert a picture to neovim highlights. If you want to convert such ANSI escape codes to neovim highlights you may refer to  [here](https://github.com/Asthestarsfalll/img2art/blob/main/img2art/utils.py#L112-L131) to write your own convert script.

**Reply by nyngwang** [reply]
@Asthestarsfalll Does it support Alacritty on macOS? (too lazy to try it, lol) Thank you

**Reply by Asthestarsfalll** [reply]
@nyngwang Yes if set config of alpha properly
